analyzer/tests/functional/ctu_failure/test_ctu_failure.py

```diff
@@ -209,7 +340,39 @@ def test_ctu_fallback(self):
         if not self.ctu_has_analyzer_display_ctu_progress:
             self.skipTest(NO_DISPLAY_CTU_PROGRESS)
 
-        output = self.__do_ctu_all(reparse=False,
+        output = self.__do_ctu_all(on_demand=False,
+                                   extra_args=[
+                                       "--verbose", "debug",
+                                       "-e", "debug.ExprInspection",
+                                       "--ctu-reanalyze-on-failure"
+                                   ])
+
+        # lib.c should be logged as its AST is loaded by Clang
+        self.assertRegex(output, r"CTU loaded AST file: .*lib\.c.ast")
```

Comment:
Does Clang really log the `.ast` postfix? That could be misleading when we have the on-demand parsing.

Reply:
I checked it, Clang just writes out the file name it loaded and in case of on-demand that should be a `.c` or `.cpp` file, isn't that so?

Reply:
Fixed this, thanks for catching! These tests are not run, because we do not have a clang that has the feature **and** otherwise passes every test as well. Would came out during the update to the next supporting Clang version, but it is better to be proactive. :+1:

Reply:
Finally fixed those as well, thanks 👍

Reply:
:+1: